app/views/groups/show.html.haml

```diff
@@ -94,7 +94,8 @@
                   .small-12.columns
                     .active_table
                       %hub.active_table_node.row.animate-repeat{id: "{{hub.hash}}",
-                      "ng-repeat" => "hub in filteredEnterprises = (Enterprises.hubs | searchEnterprises:query | taxons:activeTaxons | shipping:shippingTypes | properties:activeProperties:'distributed_properties' | orderBy:['-active', '+orders_close_at'])",
+                      "ng-repeat" => "hub in filteredEnterprises = (Enterprises.hubs | searchEnterprises:query | taxons:activeTaxons | shipping:shippingTypes | properties:activeProperties:'distributed_properties' | orderBy:['+orders_close_at'])",
+                      "ng-if" => "hub.visible",
```

Comment:
Nice solution. There's a slightly better way of doing this though:
```suggestion
                      "ng-repeat" => "hub in filteredEnterprises = (Enterprises.hubs | filter:{visible:true} | searchEnterprises:query | taxons:activeTaxons | shipping:shippingTypes | properties:activeProperties:'distributed_properties' | orderBy:['+orders_close_at'])",
```

It would be even nicer to filter on the Rails side when querying the database but that may be a lot more complicated. I'm not sure if you are up for the challenge. :wink: 

I'm happy with the above.